calico/_data/versions.yml

```diff
@@ -1,35 +1,35 @@
-- title: v3.24.0
```

Comment:
You should add a new section at the top of this file rather than replace the old section.